content/docs/how-to-guides/incremental-datasets.md

```diff
@@ -179,3 +179,20 @@ SELECT ...
 Since incremental datasets are usually timestamp based, it's a best practice to set up dataset partioning on the timestamp field to speed up downstream queries.
 
 For more information, check out the [BigQuery](warehouses/bigquery) and [Redshift](warehouses/redshift) guides.
+
+## Conditional code if incremental
+
+For functionality other than a `WHERE` clause, `when()` combined with `incremental()` can be used. For example if there was a separate field `weblogs.user_alternative_actions` containing an alternative set of user events, and it was desirable for the script to use this only when incremental, then a neat way to manage this would be to use:
+
+```js
+config { type: "incremental" }
+
+SELECT timestamp, action
+FROM ${ when(incremental(), weblogs.user_alternative_actions, weblogs.user_actions) }
+
+incremental_where {
```

Comment:
IMO we should get rid of all mention of `incremental_where` from the docs.

Reply:
Sounds good to me